test/powershell/Modules/Microsoft.PowerShell.Security/CmsMessage2.Tests.ps1

```diff
@@ -0,0 +1,180 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+using namespace System.Security.Cryptography.X509Certificates
+using namespace System.Security.Cryptography
+function New-CmsRecipient {
+  [CmdletBinding(SupportsShouldProcess = $true)]
```

Comment:
Please use 4 spaces for indentations.

Reply:
somehow my default powershell formatter uses 2 spaces, need to tweak it somehow